pyneuroml/plot/PlotMorphologyVispy.py

```diff
@@ -120,6 +125,14 @@ def create_new_vispy_canvas(
     """
     # vispy: full gl+ context is required for instanced rendering
     use(gl="gl+")
+    canvas_name = "My 3D Visualization"
+    canvas, scene_canvas, view = create_new_vispy_canvas(canvas_name)
+    canvas = vispy.app.Canvas(keys="interactive", size=(800, 600), title=canvas_name)
+    scene_canvas = SceneCanvas(keys="interactive", show=True)
+    view = scene_canvas.central_widget.add_view()
+
+    camera = scene_canvas.central_widget.camera
+    view.camera = camera
 
     canvas = scene.SceneCanvas(
```

Comment:
we're already creating a SceneCanvas, camera, view etc. here---do you need to create one too? Can the ones we already create not be used?

Reply:
yes sir you are correct I think I ignored this.